roles/ucp/templates/ucp.j2

```diff
@@ -69,6 +69,14 @@ start)
     fi
     echo fingerprint: ${fingerprint}
     echo ${fingerprint} > "{{ ucp_remote_dir }}/{{ ucp_fingerprint_file }}"
+
+    # Save the root CA Key on bootstrap node
+    # The root CA key is replicated to replicas for HA
+    /usr/bin/docker run --rm -i --name ucp \
```

Comment:
hmm, I see an issue but it is not just limited to this PR I think, we can perhaps track it as separate issue.

We use `--fresh-install` flag for service start which will cause fingerptint, instance-id and certs to be regenerated (with different values) and might break the UCP running on other nodes. `--fresh-install` was added based on Uday's suggestion to overcome some UCP re-install issues, but may be we will need a better fix from docker folks here.